kotlin/src/main/kotlin/org/axonframework/extensions/kotlin/QueryGatewayExtensions.kt

```diff
@@ -229,6 +230,37 @@ inline fun <reified R, reified Q> QueryGateway.scatterGatherOptional(query: Q, t
  * @since 0.2.0
  */
 inline fun <reified R, reified Q> QueryGateway.scatterGatherOptional(queryName: String, query: Q, timeout: Long,
-                                                                        timeUnit: TimeUnit): Stream<Optional<R>> {
+                                                                     timeUnit: TimeUnit): Stream<Optional<R>> {
     return this.scatterGather(queryName, query, ResponseTypes.optionalInstanceOf(R::class.java), timeout, timeUnit)
-}
\ No newline at end of file
+}
+
+/**
+ * Reified version of [QueryGateway.subscriptionQuery]
+ * which expects a single object as a response using [org.axonframework.messaging.responsetypes.InstanceResponseType]
+ * @param query Query to send
+ * @param Q the type of payload of the query
+ * @param I the type of initial response
+ * @param U the type of update response
+ * @return [SubscriptionQueryResult] wrapping the result of the query
+ * @see QueryGateway.subscriptionQuery
+ * @see ResponseTypes
+ * @since 0.1.0
```

Comment:
should it be changed to: "since 0.3.0" ?

Reply:
Yes please